### PR TITLE
MVP Processors Preview: Simple proxies and Proxy Places with name tokens.

### PR DESCRIPTION
**Pull request by Chris-V**

### CODE REVIEW

File: gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/processors/proxy/ProxyProcessors.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.gwtplatform.processors.tools.logger.Logger;
+import com.gwtplatform.processors.tools.outputter.Outputter;
+import com.gwtplatform.processors.tools.utils.Utils;
+
+public class ProxyProcessors {
+    private static final String NO_PROCESSORS_FOUND = "Can not find a processor for proxy `%s`.";
```
Comment (olafleur) [inline gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/processors/proxy/ProxyProcessors.java]: Cannot

File: gwtp-core/gwtp-mvp-client/src/main/resources/com/gwtplatform/mvp/Mvp.gwt.xml [hunk 2]
```diff
@@ -14,9 +14,9 @@
         <when-type-is class="com.gwtplatform.mvp.client.ApplicationController"/>
     </generate-with>
 
-    <generate-with class="com.gwtplatform.mvp.rebind.ProxyGenerator">
-        <when-type-assignable class="com.gwtplatform.mvp.client.proxy.Proxy"/>
-    </generate-with>
+    <!--<generate-with class="com.gwtplatform.mvp.rebind.ProxyGenerator">-->
```
Comment (olafleur) [inline gwtp-core/gwtp-mvp-client/src/main/resources/com/gwtplatform/mvp/Mvp.gwt.xml]: commented code?

**Comment by olafleur** [comment]
LGTM

File: gwtp-core/gwtp-mvp-client/src/main/resources/com/gwtplatform/mvp/processors/ProxyPlace.vm [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import com.gwtplatform.mvp.client.proxy.PlaceImpl;
+import com.gwtplatform.mvp.client.proxy.PlaceManager;
+import com.gwtplatform.mvp.client.proxy.ProxyImpl;
+import com.gwtplatform.mvp.client.proxy.ProxyPlaceImpl;
+import com.gwtplatform.mvp.client.proxy.RevealContentHandler;
+#printImports($imports)
+
+@Generated(value = "$processor.qualifiedName", date = "$processingDate")
+public class $impl.simpleName extends ProxyPlaceImpl<$presenterType.simpleName> implements $proxyType.simpleName {
+## TODO: This is a basically a SimpleProxy with a hard-coded name. Reuse the other file
```
Comment (meriouma) [inline gwtp-core/gwtp-mvp-client/src/main/resources/com/gwtplatform/mvp/processors/ProxyPlace.vm]: The other file?

Comment (Chris-V) [inline gwtp-core/gwtp-mvp-client/src/main/resources/com/gwtplatform/mvp/processors/ProxyPlace.vm]: SimpleProxy.vm

Comment (meriouma) [inline gwtp-core/gwtp-mvp-client/src/main/resources/com/gwtplatform/mvp/processors/ProxyPlace.vm]: Ah right. I recall that at some point we talked about removing the WrappedProxy?

**Comment by meriouma** [comment]
Maybe `$$` could be worth extracting somewhere, in case it ever needs to change?

**Pull request merged**